tests/test_strategy_storage.py

```diff
@@ -142,6 +151,9 @@ def sorted_offers(self):
         ]
         return offers[self.count]
 
+    def get_bids(self):
```

Comment:
I don't see this being used anywhere here, and this is not a property of Area class as well

Reply:
Thanks for catching that! I added these fake methods for the FakeMarket class when the two-sided market was activated by mistake, because I thought that this was the problem. Thus they should not be needed anymore, changing these